### PR TITLE
tc-331 live filtrering veilederoversikt

### PR DESCRIPTION
**Pull request by taniaholst**

### CODE REVIEW

File: src/components/veileder-checkbox-liste/veileder-checkbox-liste.tsx [hunk 1]
```diff
@@ -1,114 +1,61 @@
...
+    const veiledere: VeiledereState = useSelector((state: AppState) => state.veiledere); //SAMME SOM VALG
+    const veilederNavnQuery = useSelector((state: AppState) => state.filtreringVeilederoversikt.veilederNavnQuery);
     const [valgteElementer, setValgteElementer] = useState<string[]>([]);
-    const [open, setOpen] = useState<boolean>(props.open);
+    const form = 'veiledere';
 
     useEffect(() => {
-        setOpen(props.open);
-    }, [props.open]);
+        setValgteElementer(filtervalg.veiledere);
```
Comment (dragutin-nav) [inline src/components/veileder-checkbox-liste/veileder-checkbox-liste.tsx:24]: maybe setValgteElementer -> setValgteVeiledere

File: src/components/veileder-checkbox-liste/veileder-checkbox-liste.tsx [hunk 2]
```diff
@@ -1,114 +1,61 @@
...
-        Velg
-    </button>
-);
-
-function VeilederCheckboxListe(props: any) {
+function VeilederCheckboxListe({endreFiltervalg}: VeilederCheckboxListeProps) {
+    const filtervalg: FiltervalgModell = useSelector((state: AppState) => state.filtreringVeilederoversikt);
+    const veiledere: VeiledereState = useSelector((state: AppState) => state.veiledere); //SAMME SOM VALG
+    const veilederNavnQuery = useSelector((state: AppState) => state.filtreringVeilederoversikt.veilederNavnQuery);
     const [valgteElementer, setValgteElementer] = useState<string[]>([]);
```
Comment (dragutin-nav) [inline src/components/veileder-checkbox-liste/veileder-checkbox-liste.tsx:20]: maybe better to replace elementer with something more meaningful...

Comment (taniaholst) [reply src/components/veileder-checkbox-liste/veileder-checkbox-liste.tsx:20]: well spotted :D

File: src/filtrering/filter-konstanter.ts [hunk 3]
```diff
@@ -211,8 +211,6 @@ export const aktiviteter = {
     UTDANNINGAKTIVITET: 'Utdanning og kurs (for enslige forsørgere eller egenfinansiert)'
 };
 
-const veiledere = {};
```
Comment (dragutin-nav) [inline src/filtrering/filter-konstanter.ts:214]: I am just curious why this was here...

File: src/veiledere/veiledere-side.tsx [hunk 4]
```diff
@@ -46,6 +46,11 @@ function VeiledereSide() {
     useSetLocalStorageOnUnmount();
     LagredeFilterUIController({oversiktType: oversiktType});
 
+    const doEndreFiltervalg = (filterId: string, filterVerdi: any) => {
+        dispatch(pagineringSetup({side: 1}));
```
Comment (dragutin-nav) [inline src/veiledere/veiledere-side.tsx:50]: aha, this one will fix the issue from screen recording..

File: src/filtrering/filter-konstanter.ts (hunk 3 above)
Comment (taniaholst) [reply src/filtrering/filter-konstanter.ts:214]: i may have added it a looong time ago to check something and forgot to remove it 😅

**Review by dragutin-nav — APPROVED** [review]

**Pull request merged**